app/controllers/schools/ects/change_email_address_wizard_controller.rb

```diff
@@ -0,0 +1,54 @@
+module Schools
+  module ECTs
+    class ChangeEmailAddressWizardController < SchoolsController
+      before_action :set_ect, :set_wizard
+      before_action :reset_wizard, only: :new
+
+      FORM_KEY = :schools_ect_change_email_address_wizard
+      WIZARD_CLASS = ChangeEmailAddressWizard::Wizard
+
+      def new
+        render current_step
+      end
+
+      def create
+        if @wizard.save!
+          redirect_to @wizard.next_step_path
+        else
+          render current_step, status: :unprocessable_content
+        end
+      end
+
+    private
+
+      def set_ect
+        @ect = @school.ect_at_school_periods.find(params[:ect_id])
+      end
+
+      def set_wizard
+        @wizard = ChangeEmailAddressWizard::Wizard.new(
+          current_step:,
+          author: current_user,
+          step_params: params,
+          store:,
+          school: @school,
+          ect: @ect
+        )
+      end
+
+      def current_step
+        @current_step ||= request.path.split("/").last.underscore.to_sym.tap do |step_from_path|
+          redirect_to "/404" unless WIZARD_CLASS.step?(step_from_path)
```

Comment:
Worth adding an `as: :not_found` to the route so we can `redirect_to not_found_path`?